types/test/options.basic.ts

```diff
@@ -87,9 +87,12 @@ const options: Options = {
 };
 
 new Chart('id', {
+	type: 'bar',
 	data: {
+		labels: [],
 		datasets: [
 			{
+				data: [],
```

Comment:
@etimberg @kurkle it looks like the TS definition of the core library enforces passing `data.labels` and `data.datasets[i].data`. I don't remember if the library needs them in case of an empty chart (I though undefined values was correctly handled). And for `labels`, I'm wondering if all charts really need this input (e.g. scatter/bubble charts).